argo-cd-apps/base/member/infra-deployments/segment-bridge/segment-bridge-member.yaml

```diff
@@ -12,9 +12,13 @@ spec:
               values:
                 sourceRoot: components/segment-bridge
                 environment: staging
-                clusterDir: ""
+                clusterDir: base
           - list:
-              elements: []
+              elements:
+                - nameNormalized: stone-stg-m01
+                  values.clusterDir: stone-stg-m01
+                - nameNormalized: stone-stg-rh01
+                  values.clusterDir: stone-stg-rh01
```

Comment:
same comment as in `argo-cd-apps/base/host/segment-bridge/segment-bridge-host.yaml`

Reply:
responded above